### ISSUE TITLE
Option to disable certain features

### ISSUE THREAD

**Issue by solskogen** [midwan/amiberry#685]
I would like to be able to compile amiberry without libxml2, png, libmpeg etc at build time, for when you only need a "minimal" amiberry. The minimal option should be to only depend on SDL2.

**Comment by midwan** [comment]
`flac`, `libmpeg2` and `mpg123` are needed/used for CD audio and CD32 emulation.
`libxml2` is used for both the rp9 format and the WHDLoad loader functionality, so it's not that easy to skip out.
`libpng` is used from SDL2-image itself, so again it's necessary.

So to do what you describe, you'd need a version that has no support for CD audio and CD32, no WHDLoad-loader and rp9 - and you'd still need to keep `libpng`. I'm not sure if it's worth the trouble?

**Comment by solskogen** [comment]
I think it is, but I have no trouble understanding that this is not very
important:)

**Comment by midwan** [comment]
`libxml2` is no longer required, since the replacement with `tinyxml2` which is now part of Amiberry (as of v3.4 beta).

**Comment by midwan** [comment]
Sorry, this is not a priority for now. Perhaps we can revisit it at some point in the future, but right now I'm trying to add functionality and get it up to WinUAE standards, not remove or trim down.
